cabal.project

```diff
@@ -0,0 +1,5 @@
+packages: ./*.cabal
+source-repository-package
+  type:     git
+  location: https://github.com/awakesecurity/proto3-wire
+  tag: 8096a1e19431af603655e5c1f2488b5db907836c
```

Comment:
Stack was already using a commit pin for proto3-wire, added one for cabal. Also updated the pin in stack (this provides a `Data` instance for `FieldNumber`)